src/.env.sample

```
@@ -191,4 +191,5 @@ KAFKA_HEALTH_CHECK_TOPIC='mentoring-health-check-topic-check'
 SERVICE_NAME='MentoringService'
 
 #To disable logs for healthCheck
-HEALTH_CHECK_DEBUG_MODE='false'
\ No newline at end of file
+HEALTH_CHECK_DEBUG_MODE='false'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)